packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -60,34 +55,84 @@ export const FEE_QUOTE_SOURCES = [
 /**
  * Mainnet Curve configuration
  */
-export const MAINNET_CURVE_CONTRACTS: { [curveAddress: string]: string[] } = {
-    '0xa2b47e3d5c44877cca798226b7b8118f9bfb7a56': [
-        '0x6b175474e89094c44da98b954eedeac495271d0f', // DAI
-        '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
-    ],
-    '0x52ea46506b9cc5ef470c5bf89f17dc28bb35d85c': [
-        '0x6b175474e89094c44da98b954eedeac495271d0f', // DAI
-        '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
-        '0xdac17f958d2ee523a2206206994597c13d831ec7', // USDT
-    ],
-    '0x45f783cce6b7ff23b2ab2d70e416cdb7d6055f51': [
-        '0x6b175474e89094c44da98b954eedeac495271d0f', // DAI
-        '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
-        '0xdac17f958d2ee523a2206206994597c13d831ec7', // USDT
-        '0x0000000000085d4780b73119b644ae5ecd22b376', // TUSD
-    ],
-    '0x79a8c46dea5ada233abaffd40f3a0a2b1e5a4f27': [
-        '0x6b175474e89094c44da98b954eedeac495271d0f', // DAI
-        '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
-        '0xdac17f958d2ee523a2206206994597c13d831ec7', // USDT
-        '0x4fabb145d64652a948d72533023f6e7a623c7c53', // BUSD
-    ],
-    '0xa5407eae9ba41422680e2e00537571bcc53efbfd': [
-        '0x6b175474e89094c44da98b954eedeac495271d0f', // DAI
-        '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
-        '0xdac17f958d2ee523a2206206994597c13d831ec7', // USDT
-        '0x57ab1ec28d129707052df4df418d58a2d46d5f51', // SUSD
-    ],
+export const MAINNET_CURVE_INFOS: { [name: string]: CurveInfo } = {
+    // Busted?
+    // DaiUsdc: {
+    //     exchangeFunctionSelector: CurveFunctionSelectors.exchange_underlying,
+    //     sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy_underlying,
+    //     buyQuoteFunctionSelector: CurveFunctionSelectors.get_dx_underlying,
+    //     poolAddress: '0xa2b47e3d5c44877cca798226b7b8118f9bfb7a56',
+    //     tokens: ['0x6b175474e89094c44da98b954eedeac495271d0f', '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48'],
+    // },
+    // Busted?
+    // DaiUsdcUsdt: {
+    //     exchangeFunctionSelector: CurveFunctionSelectors.exchange_underlying,
+    //     sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy_underlying,
+    //     buyQuoteFunctionSelector: CurveFunctionSelectors.get_dx_underlying,
+    //     poolAddress: '0x52ea46506b9cc5ef470c5bf89f17dc28bb35d85c',
+    //     tokens: [
+    //         '0x6b175474e89094c44da98b954eedeac495271d0f',
+    //         '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+    //         '0xdac17f958d2ee523a2206206994597c13d831ec7',
+    //     ],
+    // },
```

Comment:
I was trying to use these in CurveBridge integration tests and they kept reverting when going USDC->DAI no matter what I did. Looking at etherscan they look pretty dead?